ThirdPartyNotices.txt

```diff
@@ -3,7 +3,7 @@ THIRD PARTY SOFTWARE NOTICES AND INFORMATION
 Do Not Translate or Localize
 
 This software incorporates material from third parties.
-Microsoft makes certain open source code available at http://3rdpartysource.microsoft.com,
```

Comment:
This is automatically generated, your change will not persist.

Reply:
@dabutvin Is there somewhere else they can make this change?

Reply:
👍  Just pushed an update to the template and atted you on it

Reply:
@dabutvin Could you provide a link to the update?